src/instructlab/model/list.py

```diff
@@ -34,6 +34,11 @@
 )
 def model_list(model_dirs: list[str], list_checkpoints: bool):
```

Comment:
```suggestion
def model_list(model_dirs: tuple[str, ...], list_checkpoints: bool):
```

Reply:
mypy fails here, the click arg has to be a list

this is more of a click issue than one we can fix. There is a sep issue for this

Reply:
oof

sure that's fine